Loading python/3.7.11-base
  ERROR: Module cannot be loaded due to a conflict.
    HINT: Might try "module unload python" first.
Loading openmpi/3.1.6-gcc11
  ERROR: Module cannot be loaded due to a conflict.
    HINT: Might try "module unload openmpi" first.
rm: cannot remove 'cp2k.out': No such file or directory
slurmstepd: error: *** JOB 4849652 ON i04r01c03s06 CANCELLED AT 2025-08-06T09:23:54 DUE TO TIME LIMIT ***
--------------------------------------------------------------------------
ORTE has lost communication with a remote daemon.

  HNP daemon   : [[2616,0],0] on node i04r01c03s06
  Remote daemon: [[2616,0],2] on node i04r01c03s08

This is usually due to either a failure of the TCP network
connection to the node, or possibly an internal failure of
the daemon itself. We cannot recover from this failure, and
therefore will terminate the job.
--------------------------------------------------------------------------
